### ISSUE TITLE
how to verify offchain attestation

### ISSUE THREAD

**Issue by 0xdwong** [ethereum-attestation-service/eas-sdk#40]
Others give me an offchain attestation. How to verify if it is valid and not modified.

**Comment by slavik0329** [comment]
@0xdwong  you can use the SDK to verify offchain attestations programmatically or you can verify it using easscan.org

**Comment by 0xdwong** [comment]
Which method in SDK should I use? I don't see any relevant information in the documentation

**Comment by slavik0329** [comment]
Yes you're right, we will have to add the verification to the documentation. If you check the tests you will see how you can use the verifyOffchainAttestationSignature method of the Offchain class to validate any offchain attestation. We will put an example up soon. Thanks!

**Comment by 0xdwong** [comment]
Ok, I will check the tests.
